src/components/Template/Bidirectionality.stories.mdx

```diff
@@ -0,0 +1,248 @@
+import { Meta, Canvas } from "@storybook/addon-docs/blocks";
+import SearchBar from "../SearchBar/SearchBar";
+import Input from "../Input/Input";
+import { InputTypes } from "../Input/InputTypes";
+import Icon from "../Icons/Icon";
+import { IconNames } from "../Icons/IconTypes";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+
+<Meta title="Bidirectionality" />
```

Comment:
There is something odd happening with the alphabetical sort order in the Storybook left menu.  Out of curiosity, I changed the title to `Template Bidirectionality` and the link jumped all the way to the bottom of the Stories list.  Are you seeing the same in your local build?

Reply:
Yes, that happens with the live-reload. If you refresh the page, it'll sort correctly. I'll change the title to `Template Bidirectionality`.

Reply:
Got it.  Looks good now.